lib/services/local/headlessauthn_watcher.go

```diff
@@ -224,6 +224,24 @@ func (h *HeadlessAuthenticationWatcher) CheckWaiter(name string) bool {
 	return false
 }
 
+// CheckWaiterStale checks if the active waiter with the given
+// headless authentication ID is marked as stale. Used in tests.
```

Comment:
I consider `// Used in tests.` an antipattern. Are there any alternative approaches that would not add to the public API solely for the purpose of a test?

Reply:
It makes the tests a lot harder to follow but I made it work :weary: I removed `CheckWaiter` as well.